### PR TITLE
Add CodeStyle fixer to the project and remove explicit PHP 5.3 compatibility

### PR DESCRIPTION
**Pull request by pablothedude**
Add extra codestyle commands to ant
https://www.pivotaltracker.com/story/show/115227481

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -90,6 +90,26 @@
...
     </target>
 
+    <target name="phpcompat"
+            description="Check code compatibilty using PHP_CodeSniffer">
+        <exec dir="${basedir}" executable="vendor/bin/phpcs" failonerror="true">
+            <arg value="--standard=ci/travis/phpcompat.xml" />
+            <arg value="--extensions=php" />
+            <arg value="--runtime-set" />
+            <arg value="testVersion" />
+            <arg value="7.0-" />
```
Comment (MKodde) [inline build.xml]: Officially we support only PHP 5.6. Although It won't be long before upgrading to 7.x. So let's leave this value at 7.0 for now?

File: ci/travis/phpcompat.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        This is used by the Ibuildings QA tools to wrap the coding standard of your choice.
+        By default it is less stringent about long lines than other coding standards
+    </description>
+
+    <exclude-pattern>src/*/*Bundle/Resources</exclude-pattern>
+    <exclude-pattern>src/*/*Bundle/Tests</exclude-pattern>
+    <exclude-pattern>src/*/Bundle/*Bundle/Resources</exclude-pattern>
+    <exclude-pattern>src/*/Bundle/*Bundle/Tests</exclude-pattern>
+    <exclude-pattern>*/Tests/*</exclude-pattern>
+
```
Comment (MKodde) [inline ci/travis/phpcompat.xml]: Have you considered adding the `tests` folder to the excludes list? The `*/Tests/*` entry could be replaced with this in my opinion.

File: build.xml (hunk 1 above)
Comment (pablothedude) [reply build.xml]: I've done this on purpose so we could see on forehand what is going to break. So all new code is 7.0 compatible. Maybe it's even better to force >7.2. What's your view on this?

File: ci/travis/phpcompat.xml (hunk 2 above)
Comment (pablothedude) [reply ci/travis/phpcompat.xml]: This is a copy/paste action from the defaul QA-tools `/ci/travis/phpcs.xml`. This is done in order support default bundle setups. When the command is called, as last parameter a folder is given and that folder is at the moment `src` or `library` so excluding `test` wouldn't be necessary.

File: build.xml (hunk 1 above)
Comment (MKodde) [reply build.xml]: I think checking for 7.2 compatibility is a good proposal. Amending this to the `build.xml`

**Pull request merged**